js/form.js

```diff
@@ -1,8 +1,9 @@
-import {picturesContainer} from './renderingPicture.js';
-import {onClickHandler, offClickHandler} from './photoScale.js';
-import {resetEffect} from './photoFilters.js';
+import {picturesContainer} from './rendering-picture.js';
+import {onClickHandler, offClickHandler,resetScalleValue} from './photo-scale.js';
+import {resetEffect} from './photo-filters.js';
```

Comment:
Либо вариант, предложеннный в main.js с отложенной инициализаций либо тут импортируем api
```suggestion
import {resetEffect} from './photo-filters.js';
import {submit} from './api.js';
```